apps/editors/tests/test_views.py

```diff
@@ -2330,6 +2330,20 @@ def test_viewing_queue(self):
         data = json.loads(r.content)
         eq_(data[str(self.addon.id)], self.editor.display_name)
 
+    def test_platform_text(self):
+        version = Version.objects.create(addon=self.addon, version='0.2')
+        version.created = datetime.today() + timedelta(days=1)
+        version.save()
+
+        for plat in (amo.PLATFORM_WIN, amo.PLATFORM_MAC):
+            File.objects.create(platform=plat.id, version=version,
+                                status=amo.STATUS_PUBLIC)
+        self.addon.update(_current_version=version)
+
+        r = self.client.get(self.url)
+        text = pq(r.content)('.editors-install').eq(1).text()
+        eq_(text, "Windows / Mac OS X")
```

Comment:
Since the switch to pytest, we try using only `assert`s: `assert text == "Windows / Mac OS X"`